### PR TITLE
DES-51 Confirm evidence request dialog

### PR DESCRIPTION
**Pull request by samtgarson**
- Homepage is 404
- Show confirmation dialog before sending a request
- Fix order of formdata when POSTing to S3

### CODE REVIEW

File: src/components/ConfirmRequestDialog.test.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          onDismiss={onDismiss}
+          onAccept={onAccept}
+          documentTypes={DocumentTypes}
+        />
+      );
+    });
+
+    it('is closed', () => {
+      expect(screen.queryByRole('dialog')).toBeVisible();
+    });
```
Comment (bogdan-zaharia) [inline src/components/ConfirmRequestDialog.test.tsx:53]: The name is "is closed" but it tests the dialog is visible, think one of them needs to change

File: src/components/ConfirmRequestDialog.test.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    it('has the correct text', () => {
+      expect(
+        screen.getByText("You're about to make a request to:")
+      ).toBeVisible();
+
+      Object.values(baseRequest.resident).forEach((text) =>
+        expect(screen.getByText(text)).toBeVisible()
+      );
+
+      expect.assertions(4);
```
Comment (bogdan-zaharia) [inline src/components/ConfirmRequestDialog.test.tsx:64]: Shouldn't we check for the exact words in the text here?

Comment (samtgarson) [reply src/components/ConfirmRequestDialog.test.tsx:64]: We are—we show a list where each bullet point is literally just the attribute of the resident, so this should be enough 
image

**Review by bogdan-zaharia — COMMENTED** [review]
Looks good, left a few comments

File: src/components/ConfirmRequestDialog.test.tsx (hunk 1 above)
Comment (samtgarson) [reply src/components/ConfirmRequestDialog.test.tsx]: Fixed

**Review by bogdan-zaharia — APPROVED** [review]
👍

**Pull request merged**